plugins/source/aws/resources/services/dynamodb/table_continuous_backups.go

```diff
@@ -19,9 +19,10 @@ func tableContinuousBackups() *schema.Table {
 			client.DefaultAccountIDColumn(false),
 			client.DefaultRegionColumn(false),
 			{
-				Name:     "table_arn",
-				Type:     arrow.BinaryTypes.String,
-				Resolver: schema.ParentColumnResolver("arn"),
+				Name:       "table_arn",
```

Comment:
Using the parent as the PK here looks good (unlike https://github.com/cloudquery/cloudquery/pull/12408#discussion_r1267911498) as it seems a table can only have a single backups description, see https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_DescribeContinuousBackups.html